src/chain.rs

```diff
@@ -18,24 +18,38 @@ use crate::{mv_memory::MvMemory, PevmTxExecutionResult};
 pub enum RewardPolicy {
     /// Ethereum
     Ethereum,
+    /// Optimism
+    #[cfg(feature = "optimism")]
+    Optimism {
+        /// L1 Fee Receipient
+        l1_fee_recipient_location_hash: crate::MemoryLocationHash,
```

Comment:
We already have `use crate::{` above so let's just add `MemoryLocationHash` there.

Reply:
Note that this part is under `#[cfg(feature = "optimism")]`.

Reply:
if we add to `use crate::{`, then we will have to add something like `#[cfg_attr(not(feature = "optimism"), allow(unused_variables))]`

Reply:
Ah that's more annoying, so yeah, let's keep this as is.